src/System.Private.CoreLib/src/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -238,7 +238,7 @@ internal ResourceSet CreateResourceSet(Stream store, Assembly assembly)
                     // the abbreviated ones emitted by InternalResGen.
                     if (CanUseDefaultResourceClasses(readerTypeName, resSetTypeName))
                     {
-                        return new RuntimeResourceSet(store);
+                        return new RuntimeResourceSet(store, permitDeserialization: true);
```

Comment:
My understanding was that we would only enable this functionality for now when WinForms sets a flag. Will that still be coming?

Reply:
I ran this by @GrabYourPitchforks and we felt that limiting the flag to assemblies which have already been loaded was a better mitigation.  If we decide that we further need a flag to opt in scenarios where we aren't loading from assemblies which have already been LoadLibraried we can add that later.